### PR TITLE
Modernize CSV plugins with v0.11-ready API/SPI

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: embulk-parser-csv/build.gradle
Comment (dmikurube) [inline embulk-parser-csv/build.gradle:119]: @sonots We're going to separate `embulk-standards` plugins as individual plugins, then to release them as independent pure-Java Embulk plugin JARs into Maven Central. In this `embulk-parser-csv`, we wanted to include you as a developer from the pull requests #434 and #440. What do you think?

Comment (sonots) [reply embulk-parser-csv/build.gradle:119]: @dmikurube I do not care either including me or not, but if this means a maintainer, probably I’m not the one.

Comment (dmikurube) [reply embulk-parser-csv/build.gradle:119]: "Maintainers" would be listed out of this parts (now in GitHub Teams). I intended this list would be just "developers who contributed a meaningful code change / an additional feature". (Just typo fixes and cosmetic changes are not included.)

**Comment by dmikurube** [comment]
`embulk-util-file` will be updated to `0.1.2` in another PR after this is merged to `master`. (#1410 needed to be on this branch to use `0.1.2`, but this PR was developed before #1410. Rebasing everything would be hard...)

**Comment by dmikurube** [comment]
`embulk-util-file:0.1.3` would work. Updated.

**Review by huylenq — APPROVED** [review]
LGTM

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**
